_docs_integrate/delete-an-attribute.md

```diff
@@ -0,0 +1,127 @@
+You can either do so by [Message]({% link _docs_integrate/data-model-overview.md %}#message) or by a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate), using the `onExistingRelationship` property of a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
+For a detailed explanation check out our guides on how to send [Requests via Messages]({% link _docs_integrate/requests-over-messages.md %}) and [Requests via RelationshipTemplates]({% link _docs_integrate/requests-over-templates.md %}).
+
+Once the peer received the Request, they can accept or reject it.
+If they want to [accept it]({% link _docs_use-cases/use-case-consumption-accept-incoming-request.md %}), they must use the [AcceptDeleteAttributeRequestItemParameters]({% link _docs_integrate/data-model-overview.md %}#acceptdeleteattributerequestitemparameters).
+Doing so, they specify a `deletionDate` on which they plan to delete the Attribute.
+In the given example, the payload would look like the following:
+
+```json
+{
+  "items": [
+    {
+      "accept": true,
+      "deletionDate": "<date the Attribute will be deleted>"
+    }
+  ]
+}
+```
+
+Now, the [enmeshed Runtime]({% link _docs_explore/61-runtime.md %}) sets the `deletionInfo` of the corresponding peer shared Attributes of the peer with `deletionStatus` `"ToBeDeleted"` and the specified `deletionDate`.
+Then, the appropriate [DeleteAttributeAcceptResponseItems]({% link _docs_integrate/data-model-overview.md %}#deleteattributeacceptresponseitem) are generated and send back in the [Reponse]({% link _docs_integrate/data-model-overview.md %}#response) to the Sender of the Request.
+There, the `deletionInfo` of the corresponding own shared Attributes is set with `deletionStatus` `"ToBeDeletedByPeer"` and the `deletionDate` received in the Response.
+
+It is also possible for the peer to [reject the Request]({% link _docs_use-cases/use-case-consumption-reject-incoming-request.md %}) for deleting Attributes as a whole, if they have a valid reason for keeping the Attributes.
+In this case, it is advised to provide a `message` using the [RejectRequestItemParameters]({% link _docs_integrate/data-model-overview.md %}#rejectrequestitemparameters), informing the Sender of the Request about the reasons not to delete the Attributes.
+{: .notice--info}
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/a738fd76-2fe0-4e3e-a0fd-bec86b3b7939" id="ZGDlhHzeKlb-"></iframe></div>
+
+## Delete peer shared Attributes
+
+The actual deletion of a peer shared Attribute happens in a separate step.
+This can either be triggered if the `deletionInfo.deletionDate` is reached, in case the deletion was requested by the owner of the Attribute, or if the peer decides they no longer need the shared Attribute.
```

Comment:
I'd reformulate: This can either be triggered if the `deletionInfo.deletionDate` is reached, in case the deletion was requested by the owner of the **peer shared Attribute**, or if the peer decides they no longer need **it**.